### DISCUSSION TITLE
Moonray for windows ?

### DISCUSSION THREAD

**Discussion by LouisV1005** [dreamworksanimation/openmoonray#2] [Q&A]
Hi community ! I'm looking on the docs etc... but I have a doubt and I would like to know if Moonray was available on windows or just on Linux for the moment ? 
Thanks for the future answers :)

**Comment by BetaGun03** [comment]
> Hi community ! I'm looking on the docs etc... but I have a doubt and I would like to know if Moonray was available on windows or just on Linux for the moment ? Thanks for the future answers :)

At the moment Moonray is only available for Docker containers and CentOs-7, you can read more information about this in their [official documentation](https://docs.openmoonray.org/getting-started/installation/building-moonray/)

**Reply by LouisV1005** [reply]
Oh ok thanks for the answer !

**Reply by BetaGun03** [reply]
You're welcome!😊
